assets/js/grp-home.js

```diff
@@ -2,9 +2,12 @@ import Vue from "vue";
 import '../css/grp-home.css';
 import { BootstrapVue, IconsPlugin } from 'bootstrap-vue';
 import FundingCyclesList from "./vue/components/FundingCyclesList";
+import filters from "./vue/utils/filters";

```

Comment:
I moved the truncate filter into a util directory, so that other apps/components can use it, So thats why we need that Vue.filter() in each of the respective JS app files